app/src/main/java/android/coding/interview/makeitawesome/networking/RestClient.java

```diff
@@ -0,0 +1,86 @@
+package android.coding.interview.makeitawesome.networking;
+
+import android.coding.interview.makeitawesome.model.Album;
+
+import com.squareup.okhttp.Cache;
+import com.squareup.okhttp.OkHttpClient;
+
+import java.io.File;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import de.halfbit.tinybus.TinyBus;
+import retrofit.Callback;
+import retrofit.RequestInterceptor;
+import retrofit.RestAdapter;
+import retrofit.RetrofitError;
+import retrofit.android.AndroidLog;
+import retrofit.client.OkClient;
+import retrofit.client.Response;
+
+/**
+ * Created by antonio on 24/07/15.
+ */
+public class RestClient {
+
+    private ApiPhoto api;
+
+
+    public static RestClient restClient= null;
+    private static final String URL = "http://jsonplaceholder.typicode.com/photos/";
+
+    protected RestClient(){
+
+        OkHttpClient okHttpClient = new OkHttpClient();
+        File cacheDir = new File(System.getProperty("java.io.tmpdir"), UUID.randomUUID().toString());
```

Comment:
System.getProperty("java.io.tmpdir") ?
Honestly, I don't know very well what it does, but that's not the way to go in Android.